fe/fe-core/src/main/java/org/apache/doris/statistics/ColumnStatistic.java

```diff
@@ -68,6 +69,7 @@ public class ColumnStatistic {
     public final double avgSizeByte;
     public final double minValue;
     public final double maxValue;
+    public final Histogram histogram;
```

Comment:
I think we'd better to place Histogram as a field of ColumnStatistic,  though it looks very natural. 
Because the calculation of Histogram is much costful than others.  We manage it in a separate structure would make 
 the framework more flexible and avoid to low down the calculation of other statistics as possible as we cam.
Above is just my personal view, I think it needs a further discussion in the community.

Reply:
I agree with your opinion. This is still a wip PR now, and I will consider using a separate histogram information collection process and structure later in the next PR.